src/App.scss

```diff
@@ -225,6 +225,10 @@ table tr:nth-child(even){
   margin-bottom: 30px;
 }
 
+.pull-right {
```

Comment:
😢

Reply:
I know, the whole page needs sorting out.